packages/networks/features.ts

```diff
@@ -17,8 +17,19 @@ export enum NetworkFeature {
   LaunchpadERC20 = "LaunchpadERC20",
   NFTMarketplaceLeaderboard = "NFTMarketplaceLeaderboard",
   CosmWasmNFTsBurner = "CosmWasmNFTsBurner",
+  CosmWasmAddressList = "CosmWasmAddressList",
 }
 
+// CosmWasm Address List
+
+const zodCCosmWasmAddressList = z.object({
+  type: z.literal(NetworkFeature.CosmWasmAddressList),
+  cwAddressListContractAddress: z.string(),
+  codeId: z.number(),
+});
```

Comment:
this feature should probably be called `NFTMarketplaceFeature`
there is no "address list feature", the address list is a tool that is used as the whitelist of the marketplace feature

we can move the other fields into it later

Reply:
Wa sorry I'm dumb today